.github/workflows/e2e_manual.yml

```diff
@@ -10,6 +10,7 @@ on:
         options:
           - genpolicy
           - getdents
+          - gpu
```

Comment:
Should this also go into nightly and/or regression test workflows?

Reply:
@katexochen do you have a stance on that? I think there should be no problem running it on nightly.

Reply:
yes, run it nightly